state/atoms/cqfmTestMRLookup.ts

```diff
@@ -0,0 +1,6 @@
+import { atom } from 'recoil';
+
+export const cqfmTestMRLookupState = atom<Record<string, fhir4.MeasureReport>>({
```

Comment:
Is there any reason this can't be a selector instead? Theoretically it's a computable value that depends on each value in the currentPatients state, the current measure/measurement period, and the desired populations on a test case. 

This should all be computable using a selector, and then you never need to use produce anywhere